src/include/pbs_share.h

```diff
@@ -44,7 +44,9 @@
 
 /* Formula special case constants */
 
-#define FORMULA_FSPERC 		"fair_share_perc"
+#define FORMULA_FSPERC 		"fairshare_perc"
+#define FORMULA_FSPERC_DEP	"fair_share_perc"
```

Comment:
You are right, but can we not do a string search for fair_share_perc in scheduler when we read the formula and log the message. I think we can also address this when we use parse job_sort_key in parse.c